src/libexpr/print.cc

```diff
@@ -1,24 +1,61 @@
-#include "print.hh"
 #include <unordered_set>
 
+#include "print.hh"
+#include "ansicolor.hh"
+#include "signals.hh"
+#include "store-api.hh"
+#include "terminal.hh"
+
 namespace nix {
 
+void pluralize(std::ostream & str, unsigned int count,
+               const std::string_view single, const std::string_view plural)
+{
+    if (count == 1)
+        str << "1 " << single;
+    else
+        str << count << " " << plural;
+}
```

Comment:
Not sure if we don't already have something like this, but if we do, it's not easy to find.
This code is prune to duplicating. Could you move it to a suitable place in `libutil`?
Perhaps a new `libutil/strings/english.hh`?